docker/legacy/Dockerfile

```diff
@@ -0,0 +1,187 @@
+# syntax=docker/dockerfile:1.2
+
+# Copyright (C) 2020 Bosch Software Innovations GmbH
+# Copyright (C) 2021 Bosch.IO GmbH
+# Copyright (C) 2021 Alliander N.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+# Set this to the version ORT should report.
+ARG ORT_VERSION="DOCKER-SNAPSHOT"
+
+# Set this to a directory containing CRT-files for custom certificates that ORT and all build tools should know about.
+ARG CRT_FILES=""
+
+# Set this to the ScanCode version to use.
+ARG SCANCODE_VERSION="3.2.1rc2"
+
+FROM adoptopenjdk/openjdk11:alpine-slim AS build
+
+# Apk install commands.
+RUN apk add --no-cache \
+        # Required for Node.js to build the reporter-web-app.
+        libstdc++ \
+        # Required to allow to download via a proxy with a self-signed certificate.
+        ca-certificates \
+        coreutils \
+        openssl
+
+COPY . /usr/local/src/ort
+
+WORKDIR /usr/local/src/ort
+
+# Gradle build.
+ARG ORT_VERSION
+RUN --mount=type=cache,target=/tmp/.gradle/ \
+    GRADLE_USER_HOME=/tmp/.gradle/ && \
+    scripts/import_proxy_certs.sh && \
+    scripts/set_gradle_proxy.sh && \
+    sed -i -r 's,(^distributionUrl=)(.+)-all\.zip$,\1\2-bin.zip,' gradle/wrapper/gradle-wrapper.properties && \
+    sed -i -r '/distributionSha256Sum=[0-9a-f]{64}/d' gradle/wrapper/gradle-wrapper.properties && \
+    ./gradlew --no-daemon --stacktrace -Pversion=$ORT_VERSION :cli:distTar :helper-cli:startScripts
+
+FROM adoptopenjdk:11-jre-hotspot-bionic
```

Comment:
I am surprised that this PR keeps the ubuntu version at bionic. That is sadly to old for my use cases. I will have to continue to maintain a fork.

Reply:
This base image also was last pushed more than a year ago. Might not be the best basis from a security point of view: https://hub.docker.com/layers/adoptopenjdk/library/adoptopenjdk/11-jre-hotspot-bionic/images/sha256-d27491ca50e6d00ca5f7331d5f2bdd136a2f8a68d57c87b9337f45a64d7f5362?context=explore

![2021-12-29_12:15:44](https://user-images.githubusercontent.com/1187050/147656795-e418fcb0-25b5-4e26-bd6f-251d54384f16.png)

Reply:
see https://github.com/oss-review-toolkit/ort/pull/4899

Reply:
@maxhbr This is the current upstream image i moved to legacy. Should i move it to new openjdk as well ?

Reply:
You should follow that, as soon as it is merged on master. But this also conflicts with the old Scancode version requiring python 3.6

Reply:
It is now merged, you could now follow that update

Reply:
I'd recommend to wait a bit, I'm planning for a few other updates to the current Docker image that we need ASAP.

Reply:
> I'd recommend to wait a bit, I'm planning for a few other updates to the current Docker image that we need ASAP.

See https://github.com/oss-review-toolkit/ort/pull/4934.